docker/Dockerfile

```diff
@@ -0,0 +1,59 @@
+### Builder
+#
+FROM golang:latest AS builder
+LABEL maintainer "John Eikenberry <jae@zhar.net>"
+
+ARG LD_FLAGS
+ARG GOTAGS
+
+WORKDIR "/go/src/github.com/hashicorp/consul-esm"
+
+COPY . .
+
+RUN ls
+
+RUN \
+  CGO_ENABLED="0" \
+  GO111MODULE=on \
+  go build -a -o "/consul-esm" \
+    -ldflags "${LD_FLAGS}" -tags "${GOTAGS}"
+
+### Final
+#
+FROM alpine:latest
+LABEL maintainer "John Eikenberry <jae@zhar.net>"
```

Comment:
Was wondering about the context for the maintainer email here instead of a hashicorp email?

Reply:
These files got included in this PR by mistake. Was doing all the repos and this was last and the one to which I paid the least specific attention as it was the same thing for the 3rd time... but it wasn't. :p

I'll fix that in a minute, but I can answer this question first. I simply thought it important to use my personal github account (and it's related email address) for everything related to the free projects maintained there. We are encouraged to use our personal github accounts for community relations and I thought this a logical extension of that.

Reply:
Docker files: Ha I see. Saw the updated commit removing the files 👍

Email: Thanks for sharing!